standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -903,13 +1033,13 @@ public Database getJDODatabase(String name) throws NoSuchObjectException {
    * @throws NoSuchObjectException
    */
   @Override
-  public boolean alterDatabase(String dbName, Database db)
+  public boolean alterDatabase(String catName, String dbName, Database db)
```

Comment:
It predates your changes, but returning boolean from this function is useless. When it fails it throws an exception anyway.